app/assess/routes.py

```diff
@@ -789,3 +791,19 @@ def application(application_id):
         assessment_status=assessment_status,
         all_uploaded_documents_section_available=fund_round.all_uploaded_documents_section_available,
     )
+
+
+@assess_bp.route(
+    "/assessor_export/<fund_short_name>/<round_short_name>/",
+    methods=["GET"],
+)
+@check_access_fund_short_name
+def assessor_export(fund_short_name: str, round_short_name: str):
+
+    fund = get_fund(fund_short_name, use_short_name=True)
+    round = get_round(fund_short_name, round_short_name, use_short_name=True)
+    export = get_applicant_export(fund.id, round.id)
```

Comment:
We don't really need to get the fund here, as `round` has it:

```suggestion
    _round = get_round(fund_short_name, round_short_name, use_short_name=True)
    export = get_applicant_export(_round.fund_id, _round.id)
```

Also, `round` is a built-in, so we should avoid shadowing it:

https://docs.python.org/3/library/functions.html#round